submissions/BlueLamantine/friends app/js/main.js

```diff
@@ -0,0 +1,329 @@
+              <div class="social-info block">
+                <span>email</span>
+                <div class="mail">
+                  <a href="mailto:${profileData.email}" class="user-mail">${profileData.email}</a>
+                </div>
+              </div>
+            </div>
+            <div class="location">
+              <div class="location-info block">
+                <span>country</span>
+                <div class="country">${profileData.location.country}</div>
+              </div>
+              <div class="location-info block">
+                <span>city</span>
+                <div class="city">${profileData.location.city}</div>
+              </div>
+            </div>
+          </div>
+          <div class="auth">
+              <span>authetication code</span>
+              <div class="code">
+                <img src="img/code.png" alt="User code" class="profile-code">
+              </div>
+          </div>
+        </div>
+      </div>
+    `;
+  }
+
+  getProfiles() {
+    this.data.forEach((result, index) => {
+      this.profilesContainer.insertAdjacentHTML(
+        'beforeend',
+        this.getHtmlProfile(result, index)
+      );
+    });
+  }
+
+  getHtmlFilter() {
+    return `   
+      <div class="filter filter-gender">
+        <p>filter by gender</p>
+        <div class="radio-wrapper">
+        <input type="radio" id="male" name="sort-gender">
+        <label for="male" class="radio-lable">Male</label>
+        <input type="radio" id="female" name="sort-gender">
+        <label for="female" class="radio-lable">female</label>
+        <input type="radio" id="allGender" name="sort-gender">
+        <label for="allGender" class="radio-lable">all</label>
+        </div>
+      </div>
+
+      <div class="filter filter-name">
+        <p>sort by name</p>
+        <div class="radio-wrapper">
+        <input type="radio" id="nameAZ" name="sort-name">
+        <label for="nameAZ" class="radio-lable">A - Z</label>
+        <input type="radio" id="nameZA" name="sort-name">
+        <label for="nameZA" class="radio-lable">Z - A</label>
+        <input type="radio" id="nameAll" name="sort-name">
+        <label for="nameAll" class="radio-lable">default</label>
+        </div>
+      </div>
+      
+      <div class="filter filter-age">
+        <p>sort by age</p>
+        <div class="radio-wrapper">
+        <input type="radio" id="age09" name="sort-age">
+        <label for="age09" class="radio-lable">young - old</label>
+        <input type="radio" id="age90" name="sort-age">
+        <label for="age90" class="radio-lable">old - young</label>
+        <input type="radio" id="ageAll" name="sort-age">
+        <label for="ageAll" class="radio-lable">default</label>
+        </div>
+      </div>
+
+      
+      <div class="filter filter-search">
+        <p>Search user by email</p>
+        <input id="nameInput" type="text" size="30" class="searchEmail icon search">
+      </div>
+    `;
+  }
+
+  getFilters() {
```

Comment:
You called function `get` but it will return nothing :)

Reply:
renamed to ```renderFilters()```